AEFeinstein/Super-2024-Swadge-FW
# Cnfs 2024

### Description

CNFS was added, SPIFFS was purged.

### Test Instructions

Compile and run firmware and emulator, ensure that the correct images are loaded

### Ticket Links

Closes #220

### Readiness Checklist
